src/test/resources/features/convertImage/image.feature

```diff
@@ -0,0 +1,283 @@
+      | width            |  10                               |
+      | gray             |  on                               |
+      | extractMetadata  |  on                               |
+      | extractThumbnail |  on                               |
+    Then I validate the response has the "200" status code
+    And I validate that the response body should match with "common/messageResponse.json" JSON schema
+    And I validate that the response container the following values
+      | status  | 200                 |
+
+  @negative
+  Scenario: Verify that an image file is not converted to a file other than image file
+    When I send a POST request to "/convertImage" with the following form data
+      | file             |  @"template/image/image.jpg"      |
+      | md5              |  1589d7f466220b7604858b1eaabdd0d2 |
+      | exportFormat     |  .mp3                             |
+      | name             |  test                             |
+      | changeSize       |  on                               |
+      | position         |  2,5                              |
+      | height           |  10                               |
+      | width            |  10                               |
+      | gray             |  on                               |
+      | extractMetadata  |  on                               |
+      | extractThumbnail |  on                               |
+    Then I validate the response has the "400" status code
+    And I validate that the response body should match with "common/errorResponse.json" JSON schema
+    And I validate that the response container the following values
+      | status  | 400                    |
+      | error   | "Invalid image format" |
+
+  @negative
+  Scenario: Verify that the file is of type image
+    When I send a POST request to "/convertImage" with the following form data
+      | file             |  @"template/video/video.mp4"      |
+      | md5              |  82d9b86b01560ce17d310f3fd2a79ca9 |
+      | exportFormat     |  .png                             |
+      | name             |  test                             |
+      | changeSize       |  on                               |
+      | position         |  2,5                              |
+      | height           |  10                               |
+      | width            |  10                               |
+      | gray             |  on                               |
+      | extractMetadata  |  on                               |
+      | extractThumbnail |  on                               |
+    Then I validate the response has the "400" status code
+    And I validate that the response body should match with "common/errorResponse.json" JSON schema
+    And I validate that the response container the following values
+      | status  | 400                         |
+      | error   | "The file isn't type image" |
+
+  @negative
+  Scenario: Verify that with invalid md5 an error displayed
```

Comment:
Please consider improving this scenario title to something like: `Verify that is not possible to perform image conversion with an invalid md5 parameter`